templates/todo/api/js/package.json

```
@@ -32,7 +32,7 @@
     "dotenv": "^16.0.1",
     "express": "^4.18.1",
     "mongodb": "^4.7.0",
-    "mongoose": "^6.3.6",
+    "mongoose": "^6.4.6",
```

```suggestion
    "mongoose": "^6.5.1",
```

I recommend updating to the latest version.